metrics/method_count.py

```diff
@@ -24,7 +26,16 @@ def _parse(self, ast, file, data):
         )
         captures = query.captures(ast.root_node)
         if len(captures) > 20:
-            data.append(["Method Count", file, 1, 1])
+            result.append(
+                Violation(
+                    "Method Count",
+                    (
+                        str(file),
+                        captures[0][0].start_point[0] + 1,
+                        captures[len(captures) - 1][0].end_point[0],
```

Comment:
What lines are reported here? The start of the first capture down to the start of the last capture? Are we sure that they are in the correct order? As in is the end_point of the last capture also the last line of the last method?

Reply:
It essentially encapsulates all captures, that is, the first method to the end of the last method. It could also just be (-1,-1) since this metric counts a violation if a file has more than 20 methods.

Reply:
I would lean towards the -1, -1 solution, because we are assuming that the last node of the list is also the last node of the file.